db/migrate/20201022143348_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+        t.belongs_to :movie, null: false
+        t.integer :rating, null: false
```

Comment:
We will also need to add a `t.belongs_to :user, null: false` if we only want signed in users to write reviews

Reply:
We are planning to add user functionality next week so I removed has_many :reviews from User model to keep consistency.